### ISSUE TITLE
User settings and workspaces no longer saved in user home folder

### ISSUE THREAD

**Issue by rwuthric** [jupyterlab/jupyterlab#13603]
## Description

This is related to issue #13597

When installing latest JupyterLab and JupyterHub (see below for versions) following [these official]( https://github.com/jupyterhub/jupyterhub-the-hard-way/blob/HEAD/docs/installation-guide-hard.md) instructions for installing JupyterHub, user settings and worksapces (the folders `user-settings` and `workspaces`) are no longer saved in the user's home folder `~/.jupyter/lab/` but rather in `/opt/jupyterhub/etc/jupyter/lab`.

As the logged user does not have write access to `/opt/jupyterhub/etc/jupyter` (which belongs to the user running JupyterHub), these files can no longer be created and result in error log entries like this:
```
[W 2022-12-15 16:02:28.893 LabApp] wrote error: "[Errno 13] Permission denied: '/opt/jupyterhub/etc/jupyter/lab'"
```
This is confirmed by running
```
> jupyter lab path
Application directory:   /opt/jupyterhub/share/jupyter/lab
User Settings directory: /opt/jupyterhub/etc/jupyter/lab/user-settings
Workspaces directory: /opt/jupyterhub/etc/jupyter/lab/workspaces
```
As effect no user settings (e.g. changing themes) or user workspaces can be saved. Only the user running the JupyterHub server can save setting changes or workspaces.

Further if all JupyterHub users will attempt to write their user settings to the same folder, this will create conflicts.

Not sure if instructions cited above must be updated to reflect newest version or if this is a bug in the new version of JupyterLab.

## Expected behavior

User settings and workspaces should be saved to a destination where the logged user has write access, as described [here](https://jupyterlab.readthedocs.io/en/stable/user/directories.html#jupyterlab-user-settings-directory)

User settings directory should point to `$HOME/.jupyter/lab/user-settings/` and workspace directory to `$HOME/.jupyter/lab/workspaces/`.

## Context

- Operating System and version: Linux Ubuntu 22.04
- Browser and version: FireFox

jupyter==1.0.0
jupyter-client==7.4.8
jupyter-console==6.4.4
jupyter-core==5.1.0
jupyter-events==0.5.0
jupyter-scheduler==1.1.4
jupyter-server==1.23.3
jupyter-server-mathjax==0.2.6
jupyter-server-terminals==0.4.2
jupyter-telemetry==0.1.0
jupyterhub==3.1.0
jupyterlab==3.5.1
jupyterlab-git==0.39.2
jupyterlab-pygments==0.2.2
jupyterlab-server==2.16.5
jupyterlab-widgets==3.0.3

**Comment by rwuthric** [comment]
Some possible workarounds are discussed here: [https://discourse.jupyter.org/t/jupyterlab-settings-dir-defaults-to-jupyterhub-root-directory/17103](https://discourse.jupyter.org/t/jupyterlab-settings-dir-defaults-to-jupyterhub-root-directory/17103)

**Comment by fcollonval** [comment]
Thanks @rwuthric 

This should be closed by https://github.com/jupyterlab/jupyterlab/pull/13589

I'm gonna release a new version 3.5.2 today with the fix.

**Comment by fcollonval** [comment]
I need to fix some error on the CI for the 3.5.x branch. So it looks the release is gonna wait for monday.
